src/css/custom.css

```diff
@@ -36,3 +36,28 @@
 .navbar__item.active {
   display: inline-block;
 }
+
+[data-theme="light"] .DocSearch {
+  --docsearch-primary-color: var(--ifm-color-primary);
+  --docsearch-text-color: var(--ifm-font-color-base);
+  --docsearch-muted-color: var(--ifm-color-secondary-darkest);
+  --docsearch-modal-background: var(--ifm-color-secondary-lighter);
+  --docsearch-searchbox-background: var(--ifm-color-secondary);
+  --docsearch-searchbox-focus-background: var(--ifm-color-white);
+  --docsearch-hit-color: var(--ifm-font-color-base);
+  --docsearch-hit-active-color: var(--ifm-color-white);
+  --docsearch-hit-background: var(--ifm-color-white);
+  --docsearch-footer-background: var(--ifm-color-white);
+}
+
+[data-theme="dark"] .DocSearch {
+  --docsearch-text-color: var(--ifm-font-color-base);
+  --docsearch-muted-color: var(--ifm-color-secondary-darkest);
+  --docsearch-modal-background: var(--ifm-background-color);
+  --docsearch-searchbox-background: var(--ifm-background-color);
+  --docsearch-searchbox-focus-background: var(--ifm-color-black);
+  --docsearch-hit-color: var(--ifm-font-color-base);
+  --docsearch-hit-active-color: var(--ifm-color-white);
+  --docsearch-hit-background: var(--ifm-color-emphasis-100);
+  --docsearch-footer-background: var(--ifm-background-surface-color);
+}
```

Comment:
Are you sure we need these, @maxdesalle? Looks like it is reapplying the defaults that should be applied by Docusaurus nonetheless?

Reply:
Yes, we do @paulrberg. The difference is subtle, but is present as you can see on the following two images. The first one is how it is with the added CSS settings, the second is without.
<img width="591" alt="CleanShot 2022-07-11 at 13 10 59@2x" src="https://user-images.githubusercontent.com/70470326/178251884-55cec8d2-fddc-440c-827e-300d88d33e5d.png">
<img width="592" alt="CleanShot 2022-07-11 at 13 11 14@2x" src="https://user-images.githubusercontent.com/70470326/178251889-5c5b8ae5-faa6-4e3f-8700-078fd91fb68a.png">

Reply:
Ok, yeah, it's a subtle difference, but noticeable.